test/jwt.test.js

```diff
@@ -1617,20 +1617,19 @@ test('errors', function (t) {
           t.equal(response.statusCode, 401)
         })
       })
-
-      t.test('authorization header format error', function (t) {
+      t.test('no bearer authorization header error', function (t) {
         t.plan(2)
 
         fastify.inject({
           method: 'get',
           url: '/verify',
           headers: {
-            authorization: 'Invalid Format'
+            authorization: 'Another Format'
```

Comment:
Keep it as `Invalid` as fine.

```suggestion
            authorization: 'Invalid Format'
```